qt/scientific_interfaces/Inelastic/Processor/ElwinPresenter.h

```diff
@@ -51,12 +52,14 @@ class MANTIDQT_INELASTIC_DLL ElwinPresenter : public DataProcessor, public IElwi
 
   void run() override;
   void setup() override;
-  bool validate() override;
```

Comment:
If possible it would be good to no longer use the `run()` and `setup()` functions inherited from the DataProcessor class. Instead of using `run()`, we should try to put all the run code directly into the `handleRun()` function. We should also try not to call the `runTab()` function because we no longer need to do the validation from within this function, or any of the rest of it. This will mean that the presenter will be less reliant on the DataProcessor class which we hope to phase out in the future.